guides/plugins/plugins/content/sitemap/add-custom-sitemap-entries.md

```diff
@@ -12,17 +12,37 @@ nav:
 Of course Shopware comes with a sitemap generation feature, including products and categories, as well as some more URLs.
 This guide however will cover how you can add your own custom URLs to the sitemap.
 
-## Prerequisites
+## By using the configuration
+
+To add a custom URL to the sitemap use the configuration setting `shopware.sitemap.custom_urls`
+
+```yaml
+shopware:
+    sitemap:
+        custom_urls:
+            -   url: 'https://www.example.com/custom-url'
```

Comment:
@M-arcus I think this needs to be a relative path instead of a full URL. At least in my local setup the hostname appeared twice

![image](https://github.com/user-attachments/assets/8f6e985f-a8cf-4a4a-87f8-cf6da24d71b2)

Reply:
You are right, I have adjusted the example.